lib/utils/getLocales/index.ts

```diff
@@ -0,0 +1,42 @@
+import { en, fr, de, ru, zh, ko, es, hi } from "make-plural/plurals";
+
+// TODO: remove NODE_ENV=test hack from package.json https://github.com/lingui/js-lingui/issues/433
+// Define locales
+interface ILocales {
+  [locale: string]: {
+    plurals: (
+      n: number | string,
+      ord?: boolean
+    ) => "zero" | "one" | "two" | "few" | "many" | "other";
+    time: string;
+    label: string;
+  };
+}
+
+const productionLocales: ILocales = {
+  en: { plurals: en, time: "en-US", label: "English" },
+  fr: { plurals: fr, time: "fr-FR", label: "Français" },
+  de: { plurals: de, time: "de-DE", label: "Deutsch" },
+  ru: { plurals: ru, time: "ru-RU", label: "Русский" },
+  "zh-CN": {
+    plurals: zh,
+    time: "zh-CN",
+    label: "中文",
+  },
+  es: { plurals: es, time: "es-ES", label: "Español" },
+  hi: { plurals: hi, time: "hi-IN", label: "हिन्दी" },
+  ko: { plurals: ko, time: "ko-KR", label: "한국어" },
```

Comment:
This is not correct IMO, korean is still only for Staging, right?
Double check with https://www.klimadao.finance/
and https://app.klimadao.finance/#/stake